src/runtime/Value.cpp

```diff
@@ -48,6 +48,44 @@ bool Value::isConstructor() const
     return true;
 }
 
+bool Value::toBoolean(ExecutionState& ec) const // $7.1.2 ToBoolean
```

Comment:
we need to make this function inline-able..
this function is used on interpreter

Reply:
Thank you for the review.
I've reverted this part to an inline function as was.